examples/01_forecasting.ipynb

```diff
@@ -613,11 +614,60 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 1.2.4 prediction intervals<a class=\"anchor\" id=\"section_1_2_4\"></a>\n",
+    "#### 1.2.4. multivariate forecasting <a class=\"anchor\" id=\"section_1_2_4\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Forecasters can also be used with multivariate datasets. Below is an example using the longley dataset from `sktime.datasets`. "
```

Comment:
I would be more verbose about the following things:
* some, not all forecasters are multivariate
* for a multivariate example: show input and output
* say that the workflows are exactly the same, except for certain things that differ
* how to see whether a forecaster is multivariate, how to find multivariate forecasters (optional?)